app/frontend/entrypoints/application.js

```diff
@@ -36,6 +36,8 @@ import "/js/TestRecordExtraction";
 import "/js/TestEnrichmentExtraction";
 import "/js/TestTransformationRecordSelector";
 import "/js/TestDestination";
+import "/js/ContentSourceFilter.js";
```

Comment:
Do you need the .js?  Super minor, just makes it consistent

Reply:
Yeah we don't need the js haha 👯